src/applications/vaos/appointment-list/components/cards/confirmed/AppointmentDateTime.jsx

```diff
@@ -25,22 +25,26 @@ export default function AppointmentDateTime({
   timezone,
   facilityId,
   twoLineFormat,
+  selectFeatureHomepageRefresh,
```

Comment:
Typically a name starting with `select` would be for a selector function, rather than a boolean. Might make sense to change this to `featureHomepageRefresh`.

Reply:
Create a new branch with only the changes needed. So this file is not modified anymore.